Submitted By: DJ Lucas (dj AT linuxfromscratch DOT org)
Date: 2009-05-14
Initial Package Version: 0.5.11
Origin: http://gitweb.freedesktop.org/?p=hal.git;a=commitdiff;h=f6af40c1e7ea54e6
Upstream Status: Applied
Description: use udevadm instead of (deprecated) udevinfo


--- a/hald/linux/blockdev.c
+++ b/hald/linux/blockdev.c
@@ -1770,7 +1770,7 @@ udev_get_device_file_for_sysfs_path (con
         char *ret;
         char *u_stdout;
         int u_exit_status;
-        const char *argv[] = {"/usr/bin/udevinfo", "--root", "--query", "name", "--path", NULL, NULL};
+        const char *argv[] = {"/sbin/udevadm", "info", "--root", "--query", "name", "--path", NULL, NULL};
         GError *g_error;
 
         ret = NULL;
--- a/hald/linux/coldplug.c
+++ b/hald/linux/coldplug.c
@@ -149,8 +149,8 @@ udev_info_to_hotplug_event (const UdevIn
 static gboolean
 hal_util_init_sysfs_to_udev_map (void)
 {
-	char *udevdb_export_argv[] = { "/usr/bin/udevinfo", "-e", NULL };
-	char *udevroot_argv[] = { "/usr/bin/udevinfo", "-r", NULL };
+	char *udevdb_export_argv[] = { "/sbin/udevadm", "info", "-e", NULL };
+	char *udevroot_argv[] = { "/sbin/udevadm", "info", "-r", NULL };
 	int udevinfo_exitcode;
 	UdevInfo *info = NULL;
 	char *p;

